arbnode/batch_poster.go

```diff
@@ -154,24 +154,25 @@ type BatchPosterConfig struct {
-	ParentChainWallet              genericconf.WalletConfig    `koanf:"parent-chain-wallet"`
-	L1BlockBound                   string                      `koanf:"l1-block-bound" reload:"hot"`
-	L1BlockBoundBypass             time.Duration               `koanf:"l1-block-bound-bypass" reload:"hot"`
-	UseAccessLists                 bool                        `koanf:"use-access-lists" reload:"hot"`
-	GasEstimateBaseFeeMultipleBips arbmath.UBips               `koanf:"gas-estimate-base-fee-multiple-bips"`
-	Dangerous                      BatchPosterDangerousConfig  `koanf:"dangerous"`
-	ReorgResistanceMargin          time.Duration               `koanf:"reorg-resistance-margin" reload:"hot"`
-	CheckBatchCorrectness          bool                        `koanf:"check-batch-correctness"`
+	ErrorDelay                                  time.Duration               `koanf:"error-delay" reload:"hot"`
+	CompressionLevel                            int                         `koanf:"compression-level" reload:"hot"`
+	DASRetentionPeriod                          time.Duration               `koanf:"das-retention-period" reload:"hot"`
+	GasRefunderAddress                          string                      `koanf:"gas-refunder-address" reload:"hot"`
+	DataPoster                                  dataposter.DataPosterConfig `koanf:"data-poster" reload:"hot"`
+	RedisUrl                                    string                      `koanf:"redis-url"`
+	RedisLock                                   redislock.SimpleCfg         `koanf:"redis-lock" reload:"hot"`
+	ExtraBatchGas                               uint64                      `koanf:"extra-batch-gas" reload:"hot"`
+	Post4844Blobs                               bool                        `koanf:"post-4844-blobs" reload:"hot"`
+	IgnoreBlobPrice                             bool                        `koanf:"ignore-blob-price" reload:"hot"`
+	ParentChainWallet                           genericconf.WalletConfig    `koanf:"parent-chain-wallet"`
+	L1BlockBound                                string                      `koanf:"l1-block-bound" reload:"hot"`
+	L1BlockBoundBypass                          time.Duration               `koanf:"l1-block-bound-bypass" reload:"hot"`
+	UseAccessLists                              bool                        `koanf:"use-access-lists" reload:"hot"`
+	GasEstimateBaseFeeMultipleBips              arbmath.UBips               `koanf:"gas-estimate-base-fee-multiple-bips"`
+	Dangerous                                   BatchPosterDangerousConfig  `koanf:"dangerous"`
+	ReorgResistanceMargin                       time.Duration               `koanf:"reorg-resistance-margin" reload:"hot"`
+	CheckBatchCorrectness                       bool                        `koanf:"check-batch-correctness"`
+	AllowPostingBatchWithOnlyBatchPostingReport bool                        `koanf:"allow-posting-batch-with-only-batch-posting-report"`
```

Comment:
Instead of a boolean, we want a "Duration" config. Treat non-empty batch as useful even if there is no useful message if they are at least that old.
By default, it should be 3 days - which normal chains won't notice, but low activity ones will have a batch once every 3 days.